src/components/RocketProfile.js

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from 'react-redux';
+
+const Joined = () => {
+  const Rockets = useSelector((state) => state.rockets.rockets);
+  const reservedRockets = Rockets.filter((rocket) => rocket.reserved === true);
+
+  return (
+    <div className="reserve-container" id="container">
+      <h1 className="profile-missions-title">My Rockets</h1>
+      <ul className="profile-missions">
+        {reservedRockets.length === 0 && <p className="no-reserve">No Rockets Reserved</p>}
+        {reservedRockets.map((rocket) => (
+          <li className="reserve-item" key={rocket.rocket_id}>
+            {rocket.rocket_name}
+          </li>
```

Comment:
- [ ] Impressive job Team. Here you have used the `map method` to map the `reservedRockets` this gets the job done and works fine. But if you think about it `map` should be used when we need to map some sort of data to another, (let's say you would like to add a salary of 5% to all the employees in your array `here map makes sense`). But the `filter method` should be used to find some sort of data that are different in some way, (Let's say you would like to find an employee whose salary is >=5000$ `here filter makes sense`). This is to say that you should use a filter instead of a map to find `reservedRockets`. 
> This is one of the main requirements we look into the projects.

Reply:
@TadesseDev Thank you for your helpful review! I just wanted to clarify that we did use the `filter` method to find the `reservedRockets` as seen in _lines 4 & 5 >_ `const Rockets = useSelector((state) => state.rockets.rockets);
  const reservedRockets = Rockets.filter((rocket) => rocket.reserved === true);` The `map` method here is used to render the `key` and `rocket_name` to the **Profile** section.  Please let me know if you think this might also be the correct way to navigate about it.

Reply:
Dear @Balikuddembe, I have noticed that you did a great job there. But my thought was it would be wise to just have all the elements and render them while filtering it. But on second thought that might affect your `No Mission Reserved` text `<p className="no-reserve">No Mission Reserved</p>`. So ill make it optional and hop you will figure it out latter.

Reply:
@TadesseDev Alright, sure thing, thank you!